deploy/openshift/templates/che-server-template.yaml

```diff
@@ -298,6 +300,9 @@ parameters:
   displayName: Eclipse Che tracing
   description: Enable or disable tracing in Eclipse Che
   value: 'false'
+- name: CHE_WSMASTER_CORS_FILTER_ENABLED
```

Comment:
I don't see the same changes in k8s deployment. Can you elaborate whether you think it is not needed to adapt it too?

Reply:
I will add k8s configuration soon

Reply:
I think @skabashnyuk started a thread on che dev mail list about the quality of PRs in Che and PRs completeness. So, he might want to have this PR complete too, e.g. support both k8s and OS deployments.

Reply:
Updated PR with K8S configuration